libs/wingsdk/src/platform/platform-manager.ts

```diff
@@ -232,9 +232,11 @@ export function _loadCustomPlatform(customPlatformPath: string): any {
     script.runInContext(context);
     return new (platformModule.exports as any).Platform();
   } catch (error) {
+    const hint = customPlatformPath.includes(".")
+      ? "Ensure the path to the platform is correct"
+      : `Ensure you have installed the platform provider by running 'npm install ${customPlatformPath}'`;
     console.error(
-      "An error occurred while loading the custom platform:",
-      error
```

Comment:
Maybe we should print the error if DEBUG=1. Otherwise it will be hard to debug issues that are not a result of uninstalled provider.

Alternatively, separate the two cases (not found and other errors).

Reply:
Good point, Ill add the log on DEBUG